Update import data documentation and rename related files for clarity

Endalaner opened the pull request:
- Renamed the import samples spreadsheet documentation to 'import-data-in-odm.md' for consistency.
- Updated references in the navigation and release notes to point to the new documentation file.
- Added comprehensive content to the new 'import-data-in-odm.md' file, detailing the data import process in ODM.
- Swapped non-informative "Note" in the Table of content with proper items for navigation
- Swapped Genestack’s customer care team in the end of the article with Genestack’s Customer Support team, enhanced with a link to a helpdesk portal

Commit:
Update import data documentation and rename related files for clarity

- Renamed the import samples spreadsheet documentation to 'import-data-in-odm.md' for consistency.
- Updated references in the navigation and release notes to point to the new documentation file.
- Added comprehensive content to the new 'import-data-in-odm.md' file, detailing the data import process in ODM.

tropnikovvl commented on docs/user-guide/doc-odm-user-guide/import-data-in-odm.md:
```diff
@@ -55,9 +55,9 @@ ODM enables you to add any data to your study. There are two main ways to associ
 
-#### NOTE
+!!! note "File size limitation"
+    Data uploading through the user interface is currently in BETA stage. The current version supports file uploading of up to 5GB. Future releases will enhance its functionality and flexibility.
```
As far as I know this limit has been removed.

Endalaner replied:
I'll just remove this part completely then. I was wondering if it still makes sense, since we changed and tested this here https://genestack.atlassian.net/browse/ODM-12031 (implemented 1.59.2).
Thanks, Vlad

eeliane replied:
@Endalaner This ticket is related to api, not to GUI, if you'd like to remove information regarding 5GB limit, please, check in advance that you can upload file with bigger size using GUI Uploader

Endalaner replied:
I have failed to import the 7gb file as a test, So I have reverted the changes in the latest commit. Thanks for the heads up guys

Commit:
Remove beta note on file size limitation in import data documentation

Commit:
minor update for the Advanced options part

eeliane commented on docs/user-guide/doc-odm-user-guide/import-data-in-odm.md:
```diff
@@ -95,33 +92,38 @@ In the subsequent dialog window, several parameters need to be defined:
 - Flow Cytometry - FACS format.
```
Please, update list of available data types to the actual state

eeliane commented on docs/user-guide/doc-odm-user-guide/import-data-in-odm.md:
```diff
@@ -95,33 +92,38 @@ In the subsequent dialog window, several parameters need to be defined:
-
-In the BETA version, the only option to link your data is through the ‘Sample Source ID’ column you must provide at the Sample (Library or Preparation tab). To link by any Sample/Library/Preparion metadata column, you’ll need to use the API.
+!!! note "Data Linking via Sample Source ID"
+    In the BETA version, the only option to link your data is through the ‘Sample Source ID’ column you must provide at the Sample (Library or Preparation tab). To link by any Sample/Library/Preparion metadata column, you’ll need to use the API.
```
This part should be rewritten to the actual behavior

eeliane commented on docs/user-guide/doc-odm-user-guide/import-data-in-odm.md:
```diff
@@ -133,7 +135,10 @@ Once the data is chosen, the upload will begin. The duration can vary from a few
+### Mandatory fields
+Please note that each uploaded data file has five mandatory read-only fields that do not belong to your template:
 
 - Genestack:accession
```
genestack:accession

eeliane commented on docs/user-guide/doc-odm-user-guide/import-data-in-odm.md:
```diff
@@ -95,33 +92,38 @@ In the subsequent dialog window, several parameters need to be defined:
+### Local or External data source
 In the second step, you’ll need to choose the file you wish to upload. There are two options:
 
 - **Local Computer**: Select the file located on your computer. Please note: this option is only available if your ODM version is coupled with AWS S3 storage (provided by default). If this option isn’t functioning properly, please get in touch with Genestack’s customer care team.
```
Looks like here Genestack’s customer care team also should be replaced with the support team. Please, fix also in the section below

Commit:
Size limitation tip returned

eeliane commented on docs/user-guide/doc-odm-user-guide/import-data-in-odm.md:
```diff
@@ -133,7 +135,10 @@ Once the data is chosen, the upload will begin. The duration can vary from a few
+After uploading, you can populate the corresponding file metadata, including the necessary details. 
+
+### Mandatory fields
+Please note that each uploaded data file has five mandatory read-only fields that do not belong to your template:
```
Could you please clarify that last three attributes are related only to tabular data files? (not to facs or vcf files)

Commit:
Enhance import data documentation with additional details and corrections

- Clarified the file size limitation note and updated the support for file formats.
- Added a new option for Flow Cytometry file format (FCS).
- Improved instructions for linking data by specifying the required template attribute.
- Updated customer support links for better accessibility.
- Included a note on mandatory fields related to tabular data files.

eeliane commented on docs/user-guide/doc-odm-user-guide/import-data-in-odm.md:
```diff
@@ -104,10 +106,7 @@ In the subsequent dialog window, several parameters need to be defined:
-
-!!! note "Data Linking via Sample Source ID"
-    In the BETA version, the only option to link your data is through the ‘Sample Source ID’ column you must provide at the Sample (Library or Preparation tab). To link by any Sample/Library/Preparion metadata column, you’ll need to use the API.
+**Link By**: Choose the ID column at the Sample (Library, Preparation) tab that will be used to identify Sample (Library, Preparation) names in the uploaded file. This column (attribute) must be in the template assigned to this study.
```
Let's clarify that for samples: signal data can be linked to the default attribute Sample Source ID or to some other template attribute, but for libraries and preparations data can be linked only to default attributes: Library ID or Preparation ID

Commit:
Revise import data documentation to emphasize upload duration and clarify data linking requirements

- Updated the note on uploading large files to highlight potential delays based on network speed.
- Clarified linking requirements for samples, libraries, and preparations in the data import process.

eeliane reviewed (approved):
Looks good
